samples/advanced-nodejs/views/math.html

```diff
@@ -176,7 +176,7 @@ <h3>Examples:</h3>
                 title: title,
                 chunks: chunks
             };
-            
```

Comment:
remove

Reply:
same comment for the other html files

---
In reply to: [292617971](https://github.com/microsoft/immersive-reader-sdk/pull/38#discussion_r292617971) [](ancestors = 292617971)